Procfile

```
@@ -0,0 +1,3 @@
+web: bundle exec puma -C config/puma.rb
+worker: bundle exec sidekiq -e production -C config/sidekiq.yml
+release: bundle exec rake db:migrate
```

```suggestion
release: bundle exec rake db:migrate
```

geralmente nao se coloca migrations no Procfile.

Cada linha aqui vira um processo que pode ser escalado em dynos no heroku

---

nevermind:

https://devcenter.heroku.com/articles/release-phase